template_language/homework/templates/movie.html

```diff
@@ -1,27 +1,28 @@
 {#  Inherit from layout #}
+{% extends 'layout.html' %}
 
-<h1 class="my-5 py-5 text-center">Avatar</h1>
-<div class="row">
-    <div class="col-md-4">
-        <img src="https://images-na.ssl-images-amazon.com/images/M/MV5BMTYwOTEwNjAzMl5BMl5BanBnXkFtZTcwODc5MTUwMw@@._V1_SX300.jpg"
-             class="card-img mb-5" alt="Avatar">
+{% block content %}
+    <h1 class="my-5 py-5 text-center">{{ movie.title }}</h1>
+    <div class="row">
+        <div class="col-md-4">
+            <img src="{{ movie.poster }}"
+                 class="card-img mb-5" alt="{{ movie.title }}">
+        </div>
+        <div class="col-md-8">
+            <p class="card-text">
+                {{ movie.plot }}
+            </p>
+            <p class="card-text">
+                <span class="font-weight-bold">Year: </span><span>{{ movie.year }}</span><br>
```

Comment:
what if you fill these spans in a cycle. Example in python:
`for value, item in dict.items()`
Google how to interpreter this to jinjia

Reply:
Done. Thanks for advice